containers/README.md

```diff
@@ -12,3 +12,48 @@ Table of Contents
  * [Issues to be Addressed](#issues)
 
 For diagrams outlining the architecture, please see the diagrams in the `docs/` folder.
+
+## Quick Start Guide
+
+This quickstart guide requires that the following are installed already:
+
+ * [Ansible](http://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)
+ * [Minishift](https://docs.openshift.org/latest/minishift/getting-started/installing.html)
+
+System Requirements:
+
+ * 10GB RAM free
+
+### Start Minishift
+
+The minishift setup requires some customization due to resource requirements. Run the following to start minishift:
+
+    minishift start --memory 10GB --cpus 4 --iso-url centos
+
+A playbook is also provided:
+
+    ansible-playbook tools/minishift-start.yml
+
+Set the context for the Openshift client to talk to minishift:
+
+    eval $(minishift oc-env)
+
+### Deploy Application
+
+Once minishift is up and running, the application can be deployed. This is done by running a playbook:
+
+    ansible-playbook foreman.yml --tags start
+
+The deployment takes a while the first time so be patient. The health and status of the deployment can be checked by running:
+
+    ./test/test.rb
+
+Once services are up and the health check returns OK for all services the application is available. To get the hostname run:
+
+    oc get routes
+
+Using the entry under HOST/PORT you can now browse to `https://<hostname>` and access the Foreman web UI.
+
+### Troubleshooting
+
+ * Sometimes when access to the UI fails, this is due to the `httpd` pod starting incorrectly. Browse the the Openshift console found at `https://$(minishift ip):8443/`. Login as `developer` with password `a` and browse to Applications > Pods. Find the httpd pod, select it, click on the Logs tab. If there is no output, browse to Applications > Deployments. Click on the httpd deployment, and click `deploy` in the upper right hand corner. Re-test the health check.
```

Comment:
I tried, but didnt find the CLI equiv command, I can do ```oc logs httpd-<id>``` to see there are no logs, any idea how to redeploy ?